apps/settings/style/icons.css

```diff
@@ -19,6 +19,7 @@ a.menu-item:after {
   right: 0.3rem;
   font-size: 25px;
   line-height: 6rem;
+  opacity: 0.7;
```

Comment:
If you can do this with `color` instead it'll be more performant. Opacity is expensive to paint.